ang/test/civiawards/dashboard/directives/more-filters-dashboard-action-button.controller.spec.js

```diff
@@ -195,6 +195,32 @@
           expect($scope.isNotificationVisible()).toBe(true);
         });
       });
+
+      describe('when filters response yields no awards types ids', () => {
+        beforeEach(() => {
+          originalCrmAPIMock = crmApiMock;
+
+          // Overwrite CRM API to return no results.
+          crmApiMock.and.returnValue($q.resolve({
+            values: []
+          }));
+        });
+
+        beforeEach(() => {
+          dialogModel.applyFilterAndCloseDialog();
+          $rootScope.$digest();
+        });
+
+        afterEach(() => {
+          crmApiMock = originalCrmAPIMock;
+        });
+
+        it('should set correct param value for case_type_id param', () => {
```

Comment:
Lets not use statement like `set correct param value`. Instead explain what that param value is.
Also lets not use technical statements like `case_type_id`. Instead the unit test statement should be a valid natural UI state(if possible), so for example
`hides all case types, cases and activities from dashboard`.

So the whole Statement including the `describe` would be

`when filters response yields no awards types ids, it hides all case types, cases and activities from dashboard`